exercises/dominoes/example.rb

```diff
@@ -0,0 +1,14 @@
+module BookKeeping
+  VERSION = 2
+end
+
+class Dominoes
+
+  def can_chain?(dominoes_list = [])
+    false
+  end
+
+  def solve(dominoes_list = [])
+
+  end
+end
```

Comment:
I think we'll need to swap your solution from `dominoes.rb` into here, and put this code into `dominoes.rb`, which is what exercism users will download to work on.